.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "etcd"]
```

Comment:
When you explained the versioning things I thought you meant there would be a submodule per version. But, there is only one submodule here. Am I misunderstanding?

Reply:
I cleaned up the PR a bit so intentions are clear (I originally deleted a large folder in this PR but that "polluted" the PR so I'll remove that folder in a different PR). Here's how this will work:

* The `scripts/load-docs.sh` script handles all versioning
* It reads the documentation versions from the `RELEASES` file
* It checks out each version in the etcd repo (`git checkout <tag>`) and copies the `Documentation` folder for that version into `content/docs/v<tag>`
* To add a new etcd version to the docs, you'll only need to add that version to the `RELEASES` file

I'm currently working on a PR that will give the `Documentation` folder the necessary structure to be processed here.

Reply:
Oh, ok, that seems reasonable.

What is the purpose of having the submodule though? Why not simply checkout the repo as part of the script? IIRC submodule is for pinning a repo to a particular version. When will we need to bump the submodule?

Reply:
Also, how will this handle information hierarchy? Will there be an index file in `content/docs/v<tag>`?

Reply:
@philips Submodule updates, in my experience, are much faster than raw clone operations when the repo is cached (as Netlify does), especially for large repos. That script runs an incremental update on every site build.

As for how the versions will be handled, each version will basically be its own self-contained “universe,” as I’ve done on other docs sites.

This is a WIP and will require some back and forth between this repo and the etcd repo.